tests/integration/integration_test_Entity.py

```diff
@@ -508,3 +505,43 @@ def test_download_file_URL_false():
     assert reupload.path == fileThatDoesntExist, "Entity should still be pointing at a URL"
     assert originalVersion == reupload.versionNumber
 
+
+#SYNPY-366
```

Comment:
Make this a docstring  and include sentence description about what the issue was